### DISCUSSION TITLE
Tower Mesh Missing Structure

### DISCUSSION THREAD

**Discussion by Vulpes-Actual** [alicevision/Meshroom#2142] [Q&A]
This is the second time I've tried to create a mesh of a cell phone tower near me and I am having difficulty getting Meshroom to actually make a good mesh from the pictures I've taken. The first attempt at scanning this tower I took the pictures a little too far away and expectedly it didn't get anything except the ground and the cabinets at the base of the tower. The second time I followed the instructions for tower mapping on the Pix4D website about their tower scanning missions and flew in the same pattern they described in the attached images. This time was a little better but I am still missing the majority of the structure, not to mention the antennas and the mounting boards. I guess my question is either: Do I need to get closer and have the object I'm scanning fill up the entire frame, or are there some settings Im missing leading up to the mesh creation? Any help is beyond appreciated!

![Capture](https://github.com/alicevision/Meshroom/assets/141360372/3132d638-e35f-4122-bcf8-94e8cc3d9ecb)

![ILL_INS_Pix4Dscan_Flight_Strategy_2](https://github.com/alicevision/Meshroom/assets/141360372/d81cb5c1-65f1-42e0-8fbc-9a1d8f56870f)

**Comment by natowi** [comment]
The example uses a Monopole Cell Tower.
I was able to find some examples with Towers, but they are not as filigree at the top like your Lattice Cell Tower:
![grafik](https://github.com/alicevision/Meshroom/assets/8138537/a98baaf3-95bf-4da4-9e9f-0594ac45bd01)
Also other projects took far more images than 137, which are probably needed, as the structure is so thin.
I am not sure if it is possible to improve the result much with more images and getting a little bit closer.
I´d try to see if other photogrammetry tools can work with your dataset and provide a better reconstruction. The result would be interesting.

**Reply by Vulpes-Actual** [reply]
I found a shorter, solid steel tower that I'm going to scan like you mentioned in the examples. Before I got tired and deleted the pictures of the last tower, I tried to up the describer density and quality but it was no use.  If it comes up with a decent mesh this time Ill find another Lattice Tower and take around 250-300 pictures, and give it another go. Do you have any other suggestions you could throw my way?

**Reply by natowi** [reply]
I think thin towers like cell towers fall under the category hard to scan. Meshroom can have difficulties, if the object of interest only fills 1/8 of the total image. 
Repetitive patterns and lack of visual features:
Meshroom will pick up far more features from the surroundings than from your object of interest, resulting in a bad reconstruction. Usually this can be mitigated by taking more and close up images (if possible, zoom in?). Masking can help in some cases, but I am not sure how well this would work with cell towers.

Limited perspectives: The images captured from the top of a lattice cell tower may have limited perspectives and may not cover all sides of the structure. This lack of comprehensive coverage can lead to incomplete or inaccurate 3D reconstructions. -> more images

Occlusions and obstructions: Lattice cell towers often have complex structures with numerous crossbeams and supports, which can obstruct the view of certain areas in the images. These occlusions can prevent key points from being captured properly, making it difficult to reconstruct the entire tower accurately. -> more images from different perspectives/ camera orientations
